model-optimizer/install_prerequisites/install_prerequisites.bat

```diff
@@ -119,34 +138,44 @@ IF %errorlevel% NEQ 0 (
 )
 
 set errorlevel=
-python "%~dp0..\mo\utils\find_ie_version.py"
+python "%ROOT_DIR%..\mo\utils\find_ie_version.py"
 IF %errorlevel% EQU 0 goto ie_search_end
 
 echo [ WARNING ] The installed OpenVINO ^(TM^) toolkit version %mo_release_version% does not work as expected. Uninstalling...
-pip3 uninstall -y openvino
+python -m pip uninstall -y openvino
 echo [ WARNING ] Consider building the Inference Engine Python API from sources
 goto ie_search_end
 
 :install_last_ov
 set errorlevel=
-pip3 install openvino
+python -m pip install openvino
 IF %errorlevel% NEQ 0 (
     echo [ WARNING ] Could not find OpenVINO ^(TM^) toolkit version available in pip for installation
     echo [ WARNING ] Consider building the Inference Engine Python API from sources
     goto ie_search_end
 )
 
 set errorlevel=
-python "%~dp0..\mo\utils\find_ie_version.py"
+python "%ROOT_DIR%..\mo\utils\find_ie_version.py"
 IF %errorlevel% EQU 0 goto ie_search_end
 
 echo [ WARNING ] The installed highest OpenVINO ^(TM^) toolkit version doesn't work as expected. Uninstalling...
-pip3 uninstall -y openvino
+python -m pip uninstall -y openvino
 echo [ WARNING ] Consider building the Inference Engine Python API from sources
 goto ie_search_end
 
 :ie_search_end
 
+IF %USE_VENV% == "true" (
+    echo.
+    echo Before running the Model Optimizer, please activate virtualenv environment by running "%VENV_DIR%\Scripts\activate.bat"
+) ELSE (
+    echo.
+    echo [ WARNING ] All Model Optimizer dependencies are installed globally.
+    echo [ WARNING ] If you want to keep Model Optimizer in separate sandbox
+    echo [ WARNING ] run install_prerequisites.sh "{caffe|tf|tf2|mxnet|kaldi|onnx}" venv
```

Comment:
install_prerequisites.sh -> install_prerequisites.bat
Please check that this command working on Windows

Reply:
The typo has been fixed. The script works fine.